modules/populateItems.ts

```diff
@@ -130,10 +143,10 @@ export const populateItems = async () => {
     .join('');
 };
 
-const generateStoreSectionData = (items) => {
-  const storeSectionData = {};
+const generateStoreSectionData = (items: Item[]): StoreSectionData => {
+  const storeSectionData: StoreSectionData = {};
 
-  items.forEach((item) => {
+  items.forEach((item: any) => {
```

Comment:
There's quite a bit of this lazy `any` typing going on... But the next task is to create an `ItemManager` class to handle mutations on the `items` table and rows inside of it. 

We have `Item` defined in the db file, which will be used in the `ItemsManager` implementation